src/Tools/README.md

```diff
@@ -23,6 +24,7 @@ dotnet watch
 dotnet user-secrets
 dotnet sql-cache
 dotnet dev-certs
+dotnet openapi
```

Comment:
Is this really going to be bundled in the .NET Core SDK? If not, this command deserves to be documented differently.

Reply:
@shirhatti now that I think about it I seem to remember you saying these wouldn't be bundled, but I'm not sure. Can we get clarification?

Reply:
I've updated this README to reflect that not everything will be shipping in the CLI